bin/config.py

```diff
@@ -64,17 +68,18 @@
     'jobs': os.cpu_count() // 2,
     'time': 600,  # Used for `bt fuzz`
     'verbose': 0,
+    'timeout': DEFAULT_TIMEOUT,
```

Comment:
The new issue is probably caused by this, since the computation for 1.5*timelimit now thinks there is a timelimit set

Reply:
Hmm, in that case, it is probably a bad idea to set the timeout with a default value. I could instead add a function like this:
```
def get_timeout(default=DEFAULT_TIMEOUT):
    return args.timeout or default
```
or i could add the `or DEFAULT_TIMEOUT` everywhere. I am not sure which is better practice.

Reply:
Either is fine

Reply:
Alternatively, you could explicitly check if the timeout argument is set for solution timeout computation